src/app/appfactory.cpp

```
@@ -20,6 +20,7 @@
 #include "framework/dockwindow/dockmodule.h"
 #include "framework/cloud/cloudmodule.h"
 #include "framework/network/networkmodule.h"
+#include "framework/update/updatemodule.h"
```

Cppcheck (2.20.0)

[information] Include file

(missingInclude)